cartoframes/dataobs.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""Data Observatory utility functions"""
+
+# Country names are pegged to the following query:
+# SELECT
+#    count(*) num_measurements,
+#    tag.key region_id,
+#    tag.value region_name
+#  FROM (
+#    SELECT *
+#    FROM OBS_GetAvailableNumerators()
+#    WHERE jsonb_pretty(numer_tags) LIKE '%subsection/%'
+#  ) numers,
+#  Jsonb_Each(numers.numer_tags) tag
+#  WHERE tag.key like 'section%'
+#  GROUP BY tag.key, tag.value
+#  ORDER BY region_name
+
+REGIONTAGS = {
```

Comment:
I agree it's not the best implementation - I plan on adding some 'catalog aware' functionality so this will be replaced soon. I like your idea of caching the result of `REGIONTAGS` with each session. For now, though, I'm going to put off the work until I can tackle it better.

Reply:
There's a ticket to track it too: https://github.com/CartoDB/cartoframes/issues/290